test/bin/test-citgm.js

```diff
@@ -2,13 +2,17 @@
 
 var test = require('tap').test;
 
-var spawn = require('../../lib/spawn');
+var spawn = require('child_process').spawn;
```

Comment:
@TheAlphaNerd Is `('child_process').spawn` not cross platform compatible? If it is, do we need `lib/spawn`? 

If it's not then the change in this PR would break things I assume.